src/js/pensions/config/form.js

```diff
@@ -113,6 +114,25 @@ function usingDirectDeposit(formData) {
   return formData['view:noDirectDeposit'] !== true;
 }
 
+function servedDuringWartime(period) {
+  const warDates = [
+    ['1916-05-09', '1917-04-05'], // Mexican Border Period (May 9, 1916 - April 5, 1917)
+    ['1917-04-06', '1918-11-11'], // World War I (April 6, 1917 - November 11, 1918)
+    ['1941-12-07', '1946-12-31'], // World War II (December 7, 1941 - December 31, 1946)
+    ['1950-06-27', '1955-01-31'], // Korean Conflict (June 27, 1950 - January 31, 1955)
+    ['1964-08-05', '1975-05-07'], // Vietnam Era (August 5, 1964 - May 7, 1975)
+    ['1990-08-02']// Gulf War (August 2, 1990)
+  ];
+  return warDates.some((warTime) => {
+    const warStart = warTime[0];
```

Comment:
You can destructure both of these:

`const [warStart, warEnd] = warTime;`
`const {from, to} = period;`